src/CppUTest/SimpleString.cpp

```diff
@@ -478,6 +478,11 @@ SimpleString HexStringFrom(long value)
     return StringFromFormat("%lx", value);
 }
 
+SimpleString HexStringFrom(int value)
+{
+    return StringFromFormat("%x", value);
+}
+
 SimpleString HexStringFrom(signed char value)
 {
     SimpleString result = StringFromFormat("%x", value);
```

Comment:
You won't notice the difference on TravisCI / Appveyor, but it will blow up the Cl2000 because it has no 8 bit width data. So char is 16 bit.